tensorflow/core/kernels/gpu_utils.h

```diff
@@ -199,6 +203,198 @@ class AutoTuneMap {
   TF_DISALLOW_COPY_AND_ASSIGN(AutoTuneMap);
 };
 
+using se::dnn::AlgorithmDesc;
+using se::dnn::AlgorithmConfig;
+template <typename Parameters>
+class AutoTuneExecutionPlanMap {
```

Comment:
How different is this vs the original autotuning map? Since it's conceptually a map from AlgorithmConfig -> AutotuningResults, I think it can be merged with the original.

Reply:
Done.